code/modules/paperwork/photography.dm

```diff
@@ -174,9 +174,9 @@
 /obj/item/device/camera/attackby(obj/item/I as obj, mob/user as mob, params)
 	if(istype(I, /obj/item/device/camera_film))
 		if(pictures_left)
-			user << "<span class='notice'>[src] still has some film in it!</span>"
+			user << "<span class='notice'>[src] still has some space left in it's card!</span>"
 			return
-		user << "<span class='notice'>You insert [I] into [src].</span>"
+		user << "<span class='notice'>You insert the [I] into [src].</span>"

```

Comment:
should be '\the', incase it's a named object
ex "You insert the Favorite Moments into the camera"